scripts/src/chartprreview/chartprreview.py

```diff
@@ -340,10 +340,14 @@ def generate_verify_report(directory, category, organization, chart, version):
 
     stdout = out.stdout.decode("utf-8")
     report_path = "report.yaml"
-    print("[INFO] report:\n", stdout)
     with open(report_path, "w") as fd:
         fd.write(stdout)
 
+    if len(out.stderr) >0 :
+        print(f'[WARNING] Errors creating report:\n{out.stderr.decode("utf-8")}')
+
```

Comment:
In case we got stderr when running the docker cmd, do still want to continue? I mean in case of failure report.yaml will contain empty file. Don't we have an error here instead of a warning?

Reply:
No, the errors typically come out if chart testing fails which is a lot, but you still get a report. If we got errors and not report that would likely be a bug in the script.